packages/stark-build/config/webpack.common.js

```diff
@@ -507,6 +509,36 @@ module.exports = options => {
 			 */
 			// TODO evaluate this
 			// new InlineManifestWebpackPlugin(),
+
+			/**
+			 * Generate html tags based on javascript maps.
+			 *
+			 * If a publicPath is set in the webpack output configuration, it will be automatically added to
+			 * href attributes, you can disable that by adding a "=href": false property.
+			 * You can also enable it to other attribute by settings "=attName": true.
+			 *
+			 * The configuration supplied is map between a location (key) and an element definition object (value)
+			 * The location (key) is then exported to the template under then htmlElements property in webpack configuration.
+			 *
+			 * Example:
+			 *  Adding this plugin configuration
+			 *  new HtmlElementsWebpackPlugin({
+			 *  headTags: { ... }
+			 *  })
+			 *
+			 * Means we can use it in the template like this:
+			 * <%= webpackConfig.htmlElements.headTags %>
+			 *
+			 * @link : https://github.com/fulls1z3/html-elements-webpack-plugin
+			 *
+			 */
+			...(fs.existsSync(helpers.root("config/head-config.common.js"))
```

Comment:
Could we change the name to something more descriptive? What about "index-head-config.js"?